ext/web/benches/timers_ops.rs

```diff
@@ -6,15 +6,6 @@ use deno_bench_util::bencher::benchmark_group;
 use deno_bench_util::bencher::Bencher;
 use deno_core::Extension;
 
-#[derive(Clone)]
-struct Permissions;
-
-impl deno_web::TimersPermission for Permissions {
```

Comment:
Ugh, I think we forgot to remove these permissions when we deprecated `--allow-hrtime` flag :(

Reply:
No this being here is intentional. Embedders must still have the ability to restrict timer granularity

Reply:
I believe this is completely gone in this PR

Reply:
Then please revert that @devsnek